swan-lake/learn/security/writing-secure-ballerina-code.md

```diff
@@ -47,14 +47,13 @@ public remote function query(@untainted string|sql:ParameterizedQuery sqlQuery,
 
 The following example constructs an SQL query with a tainted argument:
 
+Note: Taint checker is disabled by default. This can be enabled by adding --taint-check flag to build or run command
```

Comment:
Shall we say "Taint analyzer does not emit errors by default".
This is because taint analyzer always runs, but doesn't emit errors when not enabled.

Reply:
Done